net/knot/Makefile

```diff
@@ -166,6 +167,7 @@ define Build/Compile
 	$(MAKE) -C $(PKG_BUILD_DIR)/tests check-compile
 endef
 
+ifeq ($(CONFIG_arc),)
```

Comment:
Wouldn't it be better to use something like `ifeq ($(CONFIG_arc),y)`?

Reply:
Or `ifdef CONFIG_arc`